lib/plugins/create/templates/aws-go-dep/gitignore

```diff
@@ -0,0 +1,5 @@
+bin
+
+# Serverless directories
+.serverless
+vendor
```

Comment:
IMO we should document this folder as well.
Something like
```.gitignore
# Serverless directories
.serverless

# golang output binary directory
bin

# golang vendor (dependencies) directory
vendor 
```

Reply:
ah yes sure thing